pandas/util/_print_versions.py

```diff
@@ -108,7 +108,7 @@ def show_versions(as_json=False):
                 mod = importlib.import_module(modname)
             ver = ver_f(mod)
             deps_blob.append((modname, ver))
-        except:
+        except ModuleNotFoundError:
```

Comment:
Great PR. `ModuleNotFoundError` is Python 3 only, and unfortunately we need to support Python 2 few more months. Can you find a Python 2 compatible way for this please?